src/main/java/org/opentripplanner/index/transmodel/TransmodelIndexGraphQLSchema.java

```diff
@@ -2168,17 +2168,28 @@ public TransmodelIndexGraphQLSchema(Router router) {
                         .build())
                 .field(GraphQLFieldDefinition.newFieldDefinition()
                         .name("activeDates")
+                        .description(
+                            "Return a list of operating/service days a ServiceJourney run "
+                            + "on. Cancellation/replaced alterations is not included."
```

Comment:
I assume this means that if a ServiceJourney is cancelled/replaced for this date, then the date is not included in the list?